### ISSUE TITLE
Question about pretrain checkpoint and its pred res

### ISSUE THREAD

**Issue by GlennLoveZhen0124** [OpenPerceptionX/PersFormer_3DLane#70]
Hi, I have downloaded your trained model weights (Openlane benchmark), with backbone Efficientnet-B7. But when I load the weight and do inference on an openlane image, the output feature map - pred_seg_bev_map - always keep exactly the same, no matter what image I input to the model, Could you please help me to use it properly?

**Comment by RicardLee** [comment]
Hi, do you mean that the different inputs output  the same feature map?

**Comment by GlennLoveZhen0124** [comment]
> 

yes, different image output same map with exactly same values in every locations, i check the first element, and the sum of the feature map, they always keep the same

**Comment by RicardLee** [comment]
Hi, we reproduce this problem. In previous experiments, we don't evaluate the output of the seg_bev_map head. The loss of this branch doesn't converge and we believe that the mutil-task performance of PersFormer desires  more exploration.

**Comment by GlennLoveZhen0124** [comment]
Thanks for your reply ~
